apps/prairielearn/src/pages/home/home.html.tsx

```diff
@@ -29,72 +26,53 @@ export const StudentCourseSchema = z.object({
 });
 export type StudentCourse = z.infer<typeof StudentCourseSchema>;
 
+interface HomeProps {
+  resLocals: Record<string, any>;
+  instructorCourses: InstructorCourse[];
+  studentCourses: StudentCourse[];
+  adminInstitutions: Institution[];
```

Comment:
Drive-by comment: you probably want a client-safe institution type/schema here. Some things (enrollment limits and UID regexps) aren't currently exposed to anyone but global admins.

Reply:
That is now in #12731. While not strictly needed, since this page isn't hydrated, I agree it's a good idea to get that in-progress now.